messenger.rst

```diff
@@ -1675,8 +1675,34 @@ timeout                  Connection timeout. Float, value in    ``0``
                          seconds default indicates unlimited
 sentinel_master          String, if null or empty Sentinel      null
 redis_sentinel           support is disabled
+ssl                      Map of TLS options.                    null
 =======================  =====================================  =================================
 
+SSL options
+-----------
+
+The SSL options can be used change requirements for the TLS channel:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+      # config/packages/test/messenger.yaml
+      framework:
+        messenger:
```

Comment:
This node should also use 4 spaces for indentation. I forgot to add this to my previous suggestion.

Reply:
@phansys  fixed :)

Reply:
It's weird. I'm still getting the indentation with 2 spaces between `framework` and `messenger`:
```yaml
framework:
  messenger:
```
instead of:
```yaml
framework:
    messenger:
```

Reply:
I tried to fix it but got a commit conflict. See now.

Reply:
This seems fine now, thank you!